media_driver/linux/common/ddi/media_libva_caps.cpp

```diff
@@ -574,6 +584,24 @@ VAStatus MediaLibvaCaps::CheckAttribList(
     return VA_STATUS_SUCCESS;
 }
 
+VAStatus MediaLibvaCaps::GetGeneralConfigAttrib(VAConfigAttrib* attrib)
+{
+    DDI_CHK_NULL(attrib, "Null pointer", VA_STATUS_ERROR_INVALID_PARAMETER);
+
+    VAStatus status = VA_STATUS_SUCCESS;
+#if VA_CHECK_VERSION(1, 9, 0)
+    if (attrib->type == VAConfigAttribContextPriority)
+    {
+        attrib->value = CONTEXT_PRIORITY_MAX;
```

Comment:
So, what's the default value? CONTEXT_PRIORITY_MAX or CONTEXT_PRIORITY_MAX/2?
I saw set the default value as CONTEXT_PRIORITY_MAX/2 at other places.

Reply:
The code is to report the max priority to upper application. It should be CONTEXT_PRIORITY_MAX